packages/create-drupal-decoupled/src/index.ts

```diff
@@ -12,7 +12,7 @@ async function main() {
   const program = new Command()
     .name('create-drupal-decoupled') // Temporary name
     .description('Scaffold the integration with Drupal in a decoupled frontend')
-    .version('1.0.0', '-v, --version, -V', 'display the version number')
+    .version('0.1.1', '-v, --version, -V', 'display the version number')
```

Comment:
is there any way to get this information from the package.json? 🤔 so it's not hardcoded?